### PR TITLE
PROD-572: Setup sentry wizard

### PR DESCRIPTION
**Pull request by neel-ds**
- Description
Enable Sentry wizard to record full code trace

- What are the steps to test that this code is working?
NA

- Screen shots or recordings for UI changes
NA

### CODE REVIEW

**Review by marvinmarnold — COMMENTED** [review]
How does this PR achieve: "Ensure sourcemaps automatically get uploaded when code merged into prod"

**Comment by neel-ds** [comment]
> How does this PR achieve: "Ensure sourcemaps automatically get uploaded when code merged into prod"

Added additional steps in Linear

**Comment by marvinmarnold** [comment]
I'm not confident these changes will be enough. Here are some things I've observed.

- It looks like source maps have been getting uploaded: https://gator-labs.sentry.io/releases/?environment=vercel-production&project=4507686429130752&statsPeriod=14d
- 
Screenshot 2024-12-23 at 8 32 25 AM
- But when I go into [some issues](https://gator-labs.sentry.io/issues/6169908387/?environment=vercel-production&project=4507686429130752&query=is%3Aunresolved%20issue.priority%3A%5Bhigh%2C%20medium%5D&referrer=issue-stream&statsPeriod=14d&stream_index=6) on Sentry, they are not configured correctly
Screenshot 2024-12-23 at 8 46 45 AM

Maybe see if [these](https://blog.sentry.io/how-to-fix-source-map-upload-errors/) tips help.

File: sentry.client.config.ts [hunk 1]
```diff
@@ -8,18 +8,20 @@ import { ignoreErrors } from "./app/utils/sentry";
...
-
-  replaysOnErrorSampleRate: 1.0,
-
+  // Define how likely Replay events are sampled.
   // This sets the sample rate to be 10%. You may want this to be 100% while
   // in development and sample at a lower rate in production
   replaysSessionSampleRate: 0.1,
 
+  // Define how likely Replay events are sampled when an error occurs.
+  replaysOnErrorSampleRate: 1.0,
```
Comment (marvinmarnold) [inline sentry.client.config.ts:20]:
```suggestion
  replaysOnErrorSampleRate: 0.01,
```

File: sentry.client.config.ts [hunk 2]
```diff
@@ -8,18 +8,20 @@ import { ignoreErrors } from "./app/utils/sentry";
...
 
-  // Setting this option to true will print useful information to the console while you're setting up Sentry.
-  debug: false,
-
-  replaysOnErrorSampleRate: 1.0,
-
+  // Define how likely Replay events are sampled.
   // This sets the sample rate to be 10%. You may want this to be 100% while
   // in development and sample at a lower rate in production
   replaysSessionSampleRate: 0.1,
```
Comment (marvinmarnold) [inline sentry.client.config.ts:17]:
```suggestion
  replaysSessionSampleRate: 0.01,
```

**Pull request merged**